src/client/src/orangehrmRecruitmentPlugin/components/CandidateProfile.vue

```diff
@@ -0,0 +1,283 @@
+                :rules="rules.applicationDate"
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-form-row>
+          <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+            <oxd-grid-item
+              class="orangehrm-save-candidate-page --span-column-2"
+            >
+              <oxd-input-field
+                v-model="profile.notes"
+                :label="$t('general.notes')"
+                type="textarea"
+                :placeholder="$t('general.type_here')"
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-form-row>
+          <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+            <oxd-grid-item
+              class="orangehrm-save-candidate-page-full-width orangehrm-save-candidate-page-grid-checkbox"
+            >
+              <oxd-input-field
+                v-model="profile.keep"
+                type="checkbox"
+                :label="$t('recruitment.content_to_keep_data')"
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-divider />
+        <required-text></required-text>
+        <oxd-form-actions>
+          <oxd-button display-type="ghost" :label="$t('general.cancel')" />
+          <submit-button :label="$t('general.save')" />
+        </oxd-form-actions>
+      </oxd-form>
+    </div>
+  </div>
+</template>
+
+<script>
+import FullNameInput from '@/orangehrmPimPlugin/components/FullNameInput';
+import {APIService} from '@/core/util/services/api.service';
+import {
+  shouldNotExceedCharLength,
+  required,
+  validDateFormat,
+  validPhoneNumberFormat,
+  validEmailFormat,
+  maxFileSize,
+  validFileTypes,
+} from '@/core/util/validation/rules';
+import VacancyDropdown from '@/orangehrmRecruitmentPlugin/components/VacancyDropdown';
+import FileUploadInput from '@/core/components/inputs/FileUploadInput';
+import DateInput from '@/core/components/inputs/DateInput';
+import {navigate} from '@/core/util/helper/navigation';
+export default {
+  name: 'CandidateProfile',
+  components: {
+    DateInput,
+    'vacancy-dropdown': VacancyDropdown,
+    'file-upload-input': FileUploadInput,
+    'full-name-input': FullNameInput,
+  },
+  props: {
+    candidateId: {
+      type: Number,
+      required: true,
+    },
+  },
+  emits: ['getData'],
+  setup(props) {
+    const http = new APIService(
+      'https://c81c3149-4936-41d9-ab3d-e25f1bff2934.mock.pstmn.io',
+      `/recruitment/candidate/${props.candidateId}`,
+    );
+
+    return {
+      http,
+    };
+  },
+  data() {
+    return {
+      isLoading: false,
+      allowedFileTypes: [
+        'application/pdf',
+        'application/msword',
+        'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
+        'text/plain',
+        'application/rtf',
+        'application/vnd.oasis.opendocument.text',
+      ],
```

Comment:
@Super-Chama shall we move `allowedFileTypes` to BE const and get them via props

Reply:
@Ajiharan007 previously had done this with allowedFileTypes passed from BE as we have used in Admin/Job Title. But in review we noticed this screen specifically needed to support only these types so i asked to hardcode here. but yes, we can hardcode at controller level even.

Reply:
done